.github/workflows/set-announce-url.yml

```
@@ -0,0 +1,62 @@
+name: Set Announcement URL
+run-name: Set Announcement URL - ${{ inputs.announcement-url }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      announcement-url:
+        description: 'The URL of the announcement'
+        type: string
+        required: true
+
+jobs:
+  set-announce-url:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    steps:
+      - name: Validate inputs
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const announcementUrl = '${{ inputs.announcement-url }}';
+
+            if (!announcementUrl.includes('http')) {
+              throw new Error(`announcement-url must be a valid URL, got: ${announcementUrl}`);
+            }
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

```suggestion
      - name: Configure AWS credentials
        uses: aws-actions/configure-aws-credentials@v3
```

They've just released v4 recently, but let's stick with the `v3` for a while.